commands/host.vegman

```diff
@@ -289,6 +289,31 @@ function cmd_config_default {
   config_default_${action} ${yes}
 }
 
+# @sudo cmd_nmi admin
+# @restrict cmd_nmi admin
+# @doc cmd_nmi
+# Emmit NMI interrupt
```

Comment:
`Emit`. Single m. Also an 'an' article is missing, and I in NMI already stands for 'Interrupt', so:
`Emit an NMI to the host`